src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,4 @@
+.logo__image {
+  vertical-align: bottom;
+  margin: 10px 0;
+}
```

Comment:
you have one element of block logo, can't it be part of header? like `header__logo`?